modules/spotxBidAdapter.js

```diff
@@ -0,0 +1,330 @@
+import * as utils from 'src/utils';
+import { Renderer } from 'src/Renderer';
+import { registerBidder } from 'src/adapters/bidderFactory';
+import { BANNER, VIDEO } from 'src/mediaTypes';
+
+const BIDDER_CODE = 'spotx';
+const URL = '//search.spotxchange.com/openrtb/2.3/dados/';
+const ORTB_VERSION = '2.3';
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['spotx'],
+  supportedMediaTypes: [BANNER, VIDEO],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   * From Prebid.js: isBidRequestValid - Verify the the AdUnits.bids, respond with true (valid) or false (invalid).
+   *
+   * @param {object} bid The bid to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function(bid) {
+    if (bid && (typeof bid.params !== 'object' || typeof bid.params.video !== 'object')) {
```

Comment:
Why nest the parameters into a `video` property? Are there any plans to add other settings, e.g. `banner`?